test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -287,7 +287,7 @@ public static IEnumerable<object[]> MessageSizesData
                 var connection = new HubConnectionBuilder()
                         .WithUrl(new Uri(url))
                         .WithTransportType(transportType)
```

Comment:
Not related but I was wondering if we should change WithTransportType to just WithTransport

Reply:
Want me to make that change?

Reply:
I'm also wondering if With should be Use to match the rest of our stack 😄

Reply:
https://github.com/aspnet/SignalR/pull/812#discussion_r136641177 but maybe it's fine.